src/app/page.module.css

```diff
@@ -1,9 +1,36 @@
-.main {
+.mainpageContainer {
+  background: #1b1228;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+
+.welcomecontainer {
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 46.48px;
```

Comment:
delete, these are nop liness and is not doing anything to the CSS, we want to minimize dead code when possible